multiplatform-markdown-renderer/src/commonMain/kotlin/com/mikepenz/markdown/compose/components/MarkdownComponents.kt

```diff
@@ -177,15 +179,15 @@ object CurrentComponentsBridge {
     val paragraph: MarkdownComponent = {
         MarkdownParagraph(it.content, it.node, style = it.typography.paragraph)
     }
-    val orderedList: MarkdownComponent = {
-        Column(modifier = Modifier) {
-            MarkdownOrderedList(it.content, it.node, style = it.typography.ordered)
-        }
+    val orderedList: MarkdownComponent = { model ->
+        val (content, node, typography, extra) = model
+        val depth = extra[ORDERED_LIST] as? Int ?: 0
+        MarkdownOrderedList(content, node, style = typography.ordered, depth)
     }
-    val unorderedList: MarkdownComponent = {
-        Column(modifier = Modifier) {
-            MarkdownBulletList(it.content, it.node, style = it.typography.bullet)
-        }
+    val unorderedList: MarkdownComponent = { model ->
+        val (content, node, typography, extra) = model
+        val depth = extra[UNORDERED_LIST] as? Int ?: 0
+        MarkdownBulletList(content, node, style = typography.bullet, depth)
```

Comment:
I am not sure about this change as it will make those components no longer follow the same architecture as other components we handle.

Reply:
What's the key reason for this proposed change?

Reply:
> I am not sure about this change as it will make those components no longer follow the same architecture as other components we handle.

I'm still thinking of a better way to solve it

Reply:
I resolved this comment, as I had a second look and made some smaller changes to bring it closer to the rest.